deploy/helm-chart/harbor-operator/templates/instance/project.yaml

```
@@ -10,13 +10,15 @@ spec:
   parentInstance:
     name: {{ $instance.name }}
   name: {{ $project.name }}
+{{- if $project.storageLimit }}
+{{- toYaml . | nindent 2 }}
```

doesn't it have to look more like this?

```suggestion
{{- toYaml $project.storageLimit | nindent 2 }}
```

---

Good point, yet we're missing the `storageLimit` key if we apply the above. I've prepended the key in the latest commit, could you re-review?